testing/kuttl/e2e/delete/13-delete-cluster-and-check.yaml

```diff
@@ -19,6 +19,11 @@ commands:
           --output=jsonpath={.items..metadata.name}
       )
 
+      echo "DELETE: Found primary ${PRIMARY} and replica ${REPLICA} pods"
+
+      if [ -z "$PRIMARY" ]; then exit 1; fi      
+      if [ -z "$REPLICA" ]; then exit 1; fi
```

Comment:
I actually saw this test pass with replica being not found!

Reply:
Oh, yeah, as far as I could tell by watching it -- and mind, this was before the fix was implemented but after these echoes were added -- we had a double-switchover, so step 12 asserts that the switchover happened, another switchover happened but didn't complete, so we had a primary and no replica, and the test passed.

Just playing around locally, but why does ```if [[ "$(date)" < "" ]]; then echo hello; fi``` not echo?

Reply:
The comparison uses collation which can be a bit mysterious, but I suspect it comes down to the lengths. Probably "blank" is less than "not-blank", so everything is greater than the empty string.

```sh-session
$ if [[ "aa" < "a" ]]; then echo hello; fi
$ if [[ "aa" < "aa" ]]; then echo hello; fi
$ if [[ "aa" < "aaa" ]]; then echo hello; fi
hello
```